packages/mobile/src/verify/VerificationSuccessScreen.tsx

```diff
@@ -1,38 +1,57 @@
 import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
 import * as React from 'react'
 import { withNamespaces, WithNamespaces } from 'react-i18next'
-import { ScrollView, StyleSheet, Text } from 'react-native'
-import SafeAreaView from 'react-native-safe-area-view'
+import { StyleSheet, Text, View } from 'react-native'
 import { Namespaces } from 'src/i18n'
+import DancingRings from 'src/icons/DancingRings'
+import { navigate } from 'src/navigator/NavigationService'
+import { Stacks } from 'src/navigator/Screens'
 
-class VerificationSuccessScreen extends React.Component<WithNamespaces> {
-  static navigationOptions = null
+export class VerificationSuccessScreen extends React.Component<WithNamespaces> {
+  static navigationOptions = { header: null }
+
+  state = {
+    isTextWhite: false,
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      navigate(Stacks.AppStack)
+    }, 7800)
```

Comment:
Why this number? Is this the animation length? Looks like there is an `onAnimationFinish` prop: https://github.com/react-native-community/lottie-react-native/blob/master/docs/api.md#component-api

Reply:
Love it! Didn't realize that existed but should have guessed.